shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -39,6 +39,7 @@ @implementation FlutterViewController {
   BOOL _initialized;
   BOOL _viewOpaque;
   BOOL _engineNeedsLaunch;
+  NSMutableSet* _ongoingTouches;
```

Comment:
Can you please add a type to this or just use an `std::set<int64_t>` please? Thanks.

Reply:
Thanks